### ISSUE TITLE
Conditional GAN missing nn.Sigmoid() in Discriminator()

### ISSUE THREAD

**Issue by humorbeing** [eriklindernoren/PyTorch-GAN#79]
This is a good place to start to learn about GAN as pytorch user. Thank you.
I just went through conditional GAN code. I noticed these is no sigmoid at the end of discriminator network.
isn't discriminator only output number in (0,1)?

**Comment by dongdongtong** [comment]
I've also notice this point. But it may not be a bug since author use the MSE loss function for CGAN. If you must take use of the sigmoid function, you should use the BCE loss function. Good luck.

**Comment by humorbeing** [comment]
Thank you for your reply. @dongdongtong
